ow_power_system/src/PredictionHandler.cpp

```diff
@@ -0,0 +1,145 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// See PredictionHandler.h for a summary of the purpose of this file.
+
+#include <chrono>
+#include <ros/package.h>
+#include <ros/ros.h>
+#include <sensor_msgs/JointState.h>
+#include <PrognoserFactory.h>
+
+#include "PredictionHandler.h"
+
+using namespace PCOE;
```

Comment:
I'm curious where this comes from and what it stands for.

Reply:
It's part of the asynchronous prognoser example from GSAP. Interestingly, removing it doesn't seem to affect anything, so I might get rid of it.

It seems related to GSAP's prognosers; I use ``PCOE::Prognoser`` on line 105 to declare a temp prognoser.